src/layouts/CardsList/CardsList.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactElement, ReactNode } from "react";
+import styles from './CardsList.module.css';
+
+type CardsListProps = {
+    children: ReactNode
```

Comment:
**nitpick:** You don't have to declare `children` explicitly, you can use `PropsWithChildren` which is ready to use from `react` package

Reply:
Thanks for the advice, these are my first steps with typescript in react :)